rust/cyclotron-core/migrations/20240804122549_initial_job_queue_schema.sql

```diff
@@ -77,6 +77,14 @@ CREATE INDEX idx_queue_queue_name ON cyclotron_jobs(queue_name);
 -- Control tables
 ---------------------------------------------------------------------
 
+-- A simple key/value store, mostly used to let the janitor tell workers things like shard ID. This is
+-- a backchannel we should be careful to use sparingly, it just saves us from duplicating constants
+-- in N+1 config files.
```

Comment:
Hmm, we already have to configure things like the DB address per shard, I'm not sure that having these in the static config would be that bad? This also introduces a new race -- we might wait for migrations, but we probably won't wait for janitor to startup and write the latest metadata (or if we do, that seems like extra work when it could just be in a config file?).

Reply:
For example, here's cyclotron shard0 common env config:

https://github.com/PostHog/charts/blob/f0c90845e28f825c3245aee8b1d259aceff0d3c4/charts/cyclotron/templates/_helpers.tpl#L64-L76

We could have a `SHARD_ID: 0` there that's provided to every cyclotron app.

Reply:
Tore out the metadata table for now